### PR TITLE
Frontend Inheritance Organization - Day View Responsiveness Bug Fix

### PR DESCRIPTION
**Pull request by OdeChan**
Made some fixes and rearrangements to Day View to improve responsiveness

### CODE REVIEW

File: app/static/dayview.css [hunk 1]
```diff
@@ -64,46 +84,42 @@ html {
...
-  padding-left: 0.4em;
-  padding-bottom: 0.2em;
-  border: solid 0.1px var(--primary);
-  background-color: var(--borders-variant);
-  border-radius: 50px;
-  box-shadow: 1px 1px 2px #999;
+    position: fixed;
+    right: 1.2em;
+    top: 1.6em;
+    padding-right: 0.4em;
```
Comment (aviadamar) [inline app/static/dayview.css:118]: Use vars  if u can.

you can use 1 line to all paddings as such: 
```
padding: top left bottom right; 
```

File: app/static/dayview.css [hunk 2]
```diff
@@ -64,46 +84,42 @@ html {
...
+    padding-left: 0.4em;
```
Comment (aviadamar) [inline app/static/dayview.css:119]: for paddings and margins: use var if u can, if not change to rem.

File: app/static/dayview.css [hunk 3]
```diff
@@ -1,54 +1,74 @@
...
   background-color: var(--primary);
 }
 
 .schedule {
-  display: grid;
-  grid-template-rows: 1;
+    display: grid;
+    grid-template-rows: 1;
+    flex: 1;
+    margin-top: 0.65em;
```
Comment (aviadamar) [inline app/static/dayview.css:15]: Check in which plances is relevant ```em``` and in which ```rem```, now there is only use in ```em```

File: app/routers/dayview.py [hunk 4]
```diff
@@ -1,17 +1,19 @@
...
 from fastapi.templating import Jinja2Templates
+from starlette import status
 
 from app.database.models import Event, User
 from app.dependencies import get_db, TEMPLATES_PATH
-from app.routers.user import get_all_user_events
 from app.internal import zodiac
+from app.routers.user import get_all_user_events
 
 templates = Jinja2Templates(directory=TEMPLATES_PATH)
```
Comment (Gonzom) [inline app/routers/dayview.py:15]: Should be deleted. The templates from dependencies is the only one that should be used.

File: app/routers/dayview.py [hunk 5]
```diff
@@ -1,17 +1,19 @@
 from bisect import bisect_left
 from datetime import datetime, timedelta
-from typing import Tuple, Union, Optional
+from typing import Optional, Tuple, Union
 
 from fastapi import APIRouter, Depends, HTTPException, Request
 from fastapi.templating import Jinja2Templates
+from starlette import status
```
Comment (Gonzom) [inline app/routers/dayview.py:7]: Use the fastapi status, one is library here.

**Review by Gonzom — COMMENTED** [review]
Great job on cleaning up the code!
I've added a few comments to names in the files you are editing so they might be fixed while you are working on these.

File: app/static/dayview.css (hunk 1 above)
Comment (OdeChan) [reply app/static/dayview.css]: padding-top/padding-right and top/right are not the same thing

File: app/static/dayview.css [hunk 6]
```diff
@@ -72,20 +72,20 @@
...
 
-.title_size_small {
-  font-size: 0.8em;
+.title-size-small {
+  font-size: var(--text_m);
 }
 
-.title_size_Xsmall {
-  font-size: 0.4em;
+.title-size-Xsmall {
```
Comment (Gonzom) [inline app/static/dayview.css:82]: Last issue here. "title-size-Xsmall" -> "title-size-x-small", all letters should be small.

**Comment by OdeChan** [comment]
Just a quick note that @sagizaidor fixed an issue and made a PR to my repo. Now that it's been fixed and merged, this PR is ready enough for merging into develop to allow others to develop upon these changes. @yammesicka

File: app/static/dayview.css [hunk 7]
```diff
@@ -1,132 +1,166 @@
...
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
 }
-
-html {
-  font-family: 'Assistant', sans-serif;
-  text-align: center;
+#day-view {
```
Comment (ivarshav) [inline app/static/dayview.css:7]: Please add empty line before :)

File: tests/test_event.py [hunk 8]
```diff
@@ -124,14 +124,14 @@ def test_create_event_api(event_test_client, session, event):
...
-    assert b"Edit Event" in response.content
+    assert b"Event Details" in response.content
 
 
 def test_eventview_with_id(event_test_client, session, event):
     event_id = event.id
     response = event_test_client.get(f"/event/{event_id}")
     assert response.ok
-    assert b"View Event" in response.content
+    assert b"Event Details" in response.content
```
Comment (ivarshav) [inline tests/test_event.py:134]: consider extract it to a const?

**Pull request merged**